src/ol/style/icon.js

```diff
@@ -260,6 +260,16 @@ ol.style.Icon.prototype.getAnchor = function() {
 
 
 /**
+ * Get the icon color.
+ * @return {ol.Color} Color.
+ * @api
+ */
+ol.style.Icon.prototype.getColor = function() {
+  return (this.color_ && this.color_.slice) ? this.color_.slice() : this.color_;
```

Comment:
Usually our simple getters just return what was provided. So in this case, you would just always return `this.color_`.